stonesoup/sensor/sensor.py

```diff
@@ -1,5 +1,13 @@
 from abc import abstractmethod, ABC
-from typing import Set, Union, Sequence
+from functools import lru_cache
+from typing import Set, Union, Sequence, TYPE_CHECKING
+try:
+    from shapely import STRtree
+    from shapely.geometry import Polygon, Point, MultiPoint, LineString, MultiLineString
+    shapely = True
```

Comment:
```suggestion
    has_shapely = True
```
Does it make sense to define this const as something different to the name of the module for readability?